### PR TITLE
Add FAQ strings

### PR DESCRIPTION
**Pull request by codemist**
# Description

This adds the FAQ strings so that we can submit them to localizers asap. Dependent on MNTOR-1833/mozilla monitor rebrand #3934 

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: locales-pending/landing-all.ftl [hunk 1]
```diff
@@ -9,3 +9,15 @@ landing-all-hero-emailform-submit-label = Get free scan
...
 # This is a label underneath a big number "14" - it's an image that demos Monitor.
 landing-all-hero-image-chart-label = exposures
+
+# Non-US FAQ
+
+landing-all-faq-title = Frequently asked questions
+# FAQ is an acronym for Frequently Asked Questions
+landing-all-faq-see-all = See all FAQs
+landing-all-data-breach-definition-ans = What exactly is a data breach? 
+landing-all-data-breach-definition-qn = A data breach happens when personal or private information gets exposed, stolen or copied without permission. These security incidents can result from cyber attacks on websites, apps or any database where people’s personal information resides. A data breach can also happen accidentally, like if someone’s login credentials get posted publicly.
```
Comment (pdehaan) [inline locales-pending/landing-all.ftl:19]: is... this right? It feel like the `-qn` and `-ans` suffixes are reversed.

Comment (pdehaan) [inline locales-pending/landing-all.ftl:19]:
```suggestion
landing-all-data-breach-definition-qn = What exactly is a data breach? 
landing-all-data-breach-definition-ans = A data breach happens when personal or private information gets exposed, stolen or copied without permission. These security incidents can result from cyber attacks on websites, apps or any database where people’s personal information resides. A data breach can also happen accidentally, like if someone’s login credentials get posted publicly.
```

File: locales-pending/landing-premium.ftl [hunk 2]
```diff
@@ -101,3 +101,12 @@ landing-premium-plans-cards-feature-removal-plus = <b>Automatic data removal</b>
...
+landing-premium-continuous-data-removal-qn = What kinds of websites sell my personal information?
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+landing-premium-continuous-data-removal-ans = Every month, we use the information you provided about yourself (name, location and birthdate) to search across { $data_broker_sites_total_num } data broker sites that sell people’s private information. If we find your data on any of these sites, we initiate the request for removal. Data removal can take anywhere from a day to a month. This feature is available for { -brand-monitor-plus } users only. <learn_more_link>Learn more here.</learn_more_link>
```
Comment (pdehaan) [inline locales-pending/landing-premium.ftl:112]: I'm not sure `-brand-monitor-plus` is a valid term (looking at the CI failed tests). The <kbd>grep</kbd> only returns these two new results, but no definitions.

```sh
git grep "\-brand-monitor-plus" | cat

locales-pending/landing-premium.ftl:landing-premium-continuous-data-removal-ans = Every month, we use the information you provided about yourself (name, location and birthdate) to search across { $data_broker_sites_total_num } data broker sites that sell people’s private information. If we find your data on any of these sites, we initiate the request for removal. Data removal can take anywhere from a day to a month. This feature is available for { -brand-monitor-plus } users only. <learn_more_link>Learn more here.</learn_more_link>
locales-pending/premium.ftl:premium-upsell-dialog-title = Turn on automatic data removal with { -brand-monitor-plus }
```

File: locales-pending/landing-all.ftl [hunk 3]
```diff
@@ -9,3 +9,15 @@ landing-all-hero-emailform-submit-label = Get free scan
...
+landing-all-data-breach-next-steps-qn = I just found out I’m in a data breach. What do I do next?
+landing-all-data-breach-next-steps-ans = Visit { -brand-mozilla-monitor } to learn what to do after a data breach and get guided steps to resolve exposures of your personal info. Hackers rely on people reusing passwords, so it’s important to create strong, unique passwords for all your accounts. Keep your passwords in a safe place that only you have access to; this could be the same place where you store important documents or a password manager.
+landing-all-data-breach-info-qn = What information gets exposed in data breaches?
+landing-all-data-breach-info-ans = Not all breaches expose all the same info. It just depends on what hackers can access. Many data breaches expose email addresses and passwords. Others expose more sensitive information such as credit card numbers, pin numbers, and social security numbers.
```
Comment (flodolo) [inline locales-pending/landing-all.ftl:23]:
```suggestion
landing-all-data-breach-info-ans = Not all breaches expose all the same info. It just depends on what hackers can access. Many data breaches expose email addresses and passwords. Others expose more sensitive information such as credit card numbers, PIN numbers, and social security numbers.
```

File: locales-pending/landing-premium.ftl (hunk 2 above)
Comment (flodolo) [inline locales-pending/landing-premium.ftl:112]:
```suggestionlanding-premium-continuous-data-removal-ans = { $data_broker_sites_total_num ->
   *[other] Every month, we use the information you provided about yourself (name, location and birthdate) to search across { $data_broker_sites_total_num } data broker sites that sell people’s private information. If we find your data on any of these sites, we initiate the request for removal. Data removal can take anywhere from a day to a month. This feature is available for { -brand-monitor-plus } users only. <learn_more_link>Learn more here.</learn_more_link>
}
```

**Review by flodolo — APPROVED** [review]

**Review by rhelmer — APPROVED** [review]
@codemist I may not land https://github.com/mozilla/blurts-server/pull/3934 right away - feel free to pull the string changes you need and I can rebase when you land on `main` if you want. Thanks!

**Pull request merged**